datasetinsights/commands/train.py

```diff
@@ -0,0 +1,99 @@
+import logging
+
+import click
+
+import datasetinsights.constants as const
+
+logger = logging.getLogger(__name__)
+
+
+@click.command(
+    help="Start model training (and optionally validation) tasks.",
+    context_settings=const.CONTEXT_SETTINGS,
+)
+@click.option(
+    "-c",
+    "--config",
+    type=click.STRING,
+    required=True,
+    help="Path to the config estimator yaml file.",
+)
+@click.option(
+    "-p",
+    "--checkpoint-file",
+    type=click.STRING,
+    default=None,
+    help=(
+        "URI to a checkpoint file. If specified, model will load from "
+        "this checkpoint and resume training."
+    ),
+)
+@click.option(
+    "-d",
+    "--data-root",
+    type=click.STRING,
```

Comment:
Can be changed to `click.Path(exists=True)`

Reply:
fixed.